utils/doclinter.py

```diff
@@ -14,21 +14,39 @@
 from typing import List
 
 
-MAX_LINE_LENGTH = 100
+MAX_LINE_LENGTH = 90
```

Comment:
It might be better to restrict by 80 columns. But much updates is needed to do that. So I set this to 90 for now.